### PR TITLE
feat(localization): Convert remaining how to fix strings

### PR DESCRIPTION
**Pull request by sfoslund**
#### Details

Sister PR to https://github.com/microsoft/axe-windows/pull/790-- this PR covers the how to fix strings that are still outstanding after those changes. 

I don't have much context on most of these, so please feel free to correct or expand on any of these strings!

##### Motivation

Localization feature

#### Pull request checklist

- [x] Addresses an existing issue: [2002610](https://dev.azure.com/mseng/1ES/_workitems/edit/2002610)

### CODE REVIEW

File: src/Rules/Resources/HowToFix.resx [hunk 1]
```diff
@@ -118,8 +118,8 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="BoundingRectangleNotValidButOffScreen" xml:space="preserve">
-    <value>The BoundingRectangle property is not valid, but the element is off-screen.</value>
-    <comment>The description of a problem with an element currently under inspection. The term "BoundingRectangle" should not be translated (i.e. it should be treated as a loanword).</comment>
+    <value>Consider removing the BoundingRectangle property when the element is off-screen.</value>
```
Comment (ferBonnin) [inline src/Rules/Resources/HowToFix.resx:121]: we should be using imperative verbs: Remove the Bounding...

File: src/Rules/Resources/HowToFix.resx [hunk 2]
```diff
@@ -166,8 +166,8 @@
     <comment>The description of a problem with an element currently under inspection. The terms "IsKeyboardFocusable" and "IsEnabled" should not be translated (i.e. they should be treated as loanwords).</comment>
   </data>
   <data name="IsKeyboardFocusableForListItemShouldBeTrue" xml:space="preserve">
-    <value>The IsKeyboardFocusable property for the given list item is false, but the element has children that are focusable. The element should probably be focusable instead of its children.</value>
-    <comment>The description of a problem with an element currently under inspection. The term "IsKeyboardFocusable" should not be translated (i.e. it should be treated as a loanword).</comment>
+    <value>Consider making this element focusable instead of its children.</value>
```
Comment (ferBonnin) [inline src/Rules/Resources/HowToFix.resx:169]: imperative

**Review by ferBonnin — CHANGES_REQUESTED** [review]
For how to fix, we should use imperative verbs such as  "Provide," "Remove," or "Use."

File: src/Rules/Resources/HowToFix.resx [hunk 3]
```diff
@@ -130,16 +130,16 @@
     <comment>The description of a problem with an element currently under inspection. The term "LocalizedControlType" should not be translated (i.e. it should be treated as a loanword).</comment>
   </data>
   <data name="ComboBoxShouldNotSupportScrollPattern" xml:space="preserve">
-    <value>A combo box should not support the Scroll pattern. This rule may be reported as a warning because some platforms have combo boxes that support the Scroll pattern by default, which app developers can't easily fix.</value>
-    <comment>The description of a problem with an element currently under inspection. The term "Scroll" should not be translated (i.e. it should be treated as a loanword).</comment>
+    <value>Modify the combo box such that is does not support the Scroll pattern.</value>
```
Comment (DaveTryon) [inline src/Rules/Resources/HowToFix.resx:133]: Should the verbiage here match what we do in other strings? Maybe
```suggestion
    <value>Modify the combo box to not support the Scroll pattern.</value>
```

**Review by DaveTryon — APPROVED** [review]
One minor suggestion, otherwise looks good to me

**Review by ferBonnin — APPROVED** [review]

**Pull request merged**